src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -252,7 +252,7 @@ BEGIN TRANSACTION
                         COMMIT TRANSACTION
     
                         -- result set 5
-                        SELECT Value, SystemId from dbo.System;
+                        SELECT TOP 10000 Value, SystemId from dbo.System;
```

Comment:
Agree it won't crash, but will this still result in more bugs since data will be missing?

Reply:
Thanks for the comment! I was going to reach out to you for discussion! :D 
Yes, it can. 

I have some options in mind: 
* We limit this query and unblock the account failing in prod. 
* We add to our backlog the dynamic load into memory.

In my mind, having this info added dynamically in memory makes more sense, but we need to unblock a prod account to minimize IcMs. 

From the discussion with the team, having more than 10k resources in dbo.System does not sound correct. Even our biggest customers don't have that number of namespaces.